libethereum/EthereumHost.h

```diff
@@ -140,6 +141,9 @@ class EthereumHost: public p2p::HostCapability<EthereumPeer>, Worker
     std::shared_ptr<EthereumHostDataFace> m_hostData;
     std::shared_ptr<EthereumPeerObserverFace> m_peerObserver;
 
+    std::shared_ptr<std::random_device> m_randomSeed;
```

Comment:
We don't to share them with anyone, so we don't need the overhead of `shared_ptr`, so make them just value members (`std::random_device m_randomDevice;`)